website_sale_custom_filter/controllers/main.py

```diff
@@ -0,0 +1,119 @@
+# Copyright 2022 Camptocamp
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+from odoo.http import request
+
+from odoo.addons.website_sale.controllers.main import WebsiteSale
+
+
+class WebsiteSale(WebsiteSale):
+
+    # get min & max from url params for range widgets only
+    def get_min_max_custom_filters(self):
+        req = dict(request.httprequest.args)
+        min_max_filter_vals = {}
+        if req.get("min_cust_filter", False):
+            for vals in req["min_cust_filter"].split(","):
+                f_id, f_min_val = vals.split("_")
+                f_min_val = int(f_min_val)
+                if not min_max_filter_vals.get(f_id, False):
+                    min_max_filter_vals[f_id] = {"min": f_min_val}
+                else:
+                    min_max_filter_vals[f_id]["min"] = f_min_val
```

Comment:
```suggestion
            for vals in req.get("min_cust_filter", "").split(","):
                f_id, f_min_val = vals.split("_")
                f_min_val = int(f_min_val)
                if not min_max_filter_vals.get(f_id, False):
                    min_max_filter_vals[f_id] = {"min": f_min_val}
                else:
                    min_max_filter_vals[f_id]["min"] = f_min_val
```
and you can indent back everything by one tab, same for the block below

Reply:
afaics in worst case this will return list with empty string which gets evaluated as a True so here i also need to add if-cond. to be sure vals is True (non-empty string), otherwise we end up with code trying to process this string.
```
(Pdb) req.get("min_cust_filter","").split(",")
['']
(Pdb) bool(req.get("min_cust_filter","").split(","))
True